sdks/python/apache_beam/metrics/metric_test.py

```diff
@@ -159,15 +163,26 @@ def process(self, element):
     res = pipeline.run()
     res.wait_until_finish()
 
+    # Verify static counter.
+    metric_results = (
+        res.metrics().query(
+            MetricsFilter().with_metric(SomeDoFn.static_counter_elements)))
+    outputs_static_counter = metric_results['counters'][0]
+
+    self.assertEqual(
+        outputs_static_counter.key.metric.name,
+        'metrics_static_counter_element')
+    self.assertEqual(outputs_static_counter.committed, 8)
+
     # Verify user counter.
     metric_results = (
         res.metrics().query(
             MetricsFilter().with_name('metrics_user_counter_element')))
-    outputs_counter = metric_results['counters'][0]
+    outputs_user_counter = metric_results['counters'][0]
```

Comment:
Is this the canonical way of fetching a metric result?
Depending on if inc() is ever triggered, or triggered in different Steps, there can be 0 to many entries in the counters list.

Reply:
you're right. I would not call this the canonical way. Though tbh this API should be improved to be less JAva-like. 

In this case, we query it this way only for purposes of the unittest, because we know what to expect.